config/sync/views.view.recipe_list.yml

```diff
@@ -383,6 +383,33 @@ display:
       tags:
         - 'config:field.storage.node.field_description'
         - 'config:field.storage.node.field_image'
+  block_1:
```

Comment:
This configuration still contains stuff that were fixed by Gabor.

Should this PR contain any configuration except for the module and the `core.extension.yml` file?

Reply:
Yes, I had to fix that block because it didn't appear as it should have (and it's too small for a new PR). But just in case I re-installed my local site and changed only the title, then updated this PR, so now it's sure I haven't ruin Gábor's work.